docs/source/Quickstart/04_non-virt.md

```diff
@@ -10,6 +10,7 @@ In order to do so, some steps should be done beforehand.
 
 ### 1. Get the project and its dependencies
 ```Bash
+$ cd $HOME
```

Comment:
actually we don't really have to be in $HOME - any location where the current user can write would be OK.

Reply:
Not really? though that's a detail. Linking to the couple of parameters we can set in order to "move" locations is possible - let's go with your PR, and we can iterate with proper links and references.